plot_pixel_values.py

```diff
@@ -78,26 +142,54 @@ def get_opt():
         if i in i_brightest:
             cols.append(dat[colname])
 
-    for im in range(n_fig):
-        fig = figs[im]
-        axes = axess[im]
-        i_col = 0
-        for r in range(N):
-            for c in range(N):
-                ax = axes[r][c]
-                x = dat['dt']
-                y = cols[i_col]
-                i_col += 1
-                if ax.lines:
-                    l0 = ax.lines[0]
-                    l0.set_data(x, y)
-                    ax.relim()
-                    ax.autoscale_view()
-                else:
-                    ax.plot(x, y)
-
-        plt.draw()
-        fig.canvas.draw()
-        fig.canvas.flush_events()
+    i_col = 0
+    fits = {}
+    for r in range(N):
+        for c in range(N):
+            ax = axes[r][c]
+            x = dat['dt']
+            y = cols[i_col]
+            i_col += 1
+            if ax.lines:
+                l0 = ax.lines[0]
+                l0.set_data(x, y)
+                ax.relim()
+                ax.autoscale_view()
+            else:
+                ax.plot(x, y)
+            ax.texts = []
+            ax.annotate("{}".format(y.name),
+                        xy=(0.5, 0.5), xycoords="axes fraction",
+                        ha='center', va='center',
+                        color='lightgrey')
+            if opt.fit_scaling:
+                # Use only non-zero pixel data for fits
+                nonzero = y != 0
```

Comment:
Put a (frozen) after it if that's convenient.  We'll know in any case as you said from the value.